packages/ubuntu_bootstrap/lib/installer.dart

```diff
@@ -172,16 +173,16 @@ Future<void> runInstallerApp(
     final flavorService = await FlavorService.load();
     tryRegisterService<FlavorService>(() => flavorService);
 
-    final endpoint = await initialized;
-    await _initInstallerApp(endpoint);
-
     runApp(ProviderScope(
       child: _InstallerApp(
         theme: theme,
         darkTheme: darkTheme,
         themeVariant: themeVariant,
         windowTitle: windowTitle,
         flavor: flavorService.flavor,
+        init: getService<SubiquityServer>()
+            .start(args: subiquityArgs)
```

Comment:
I've moved this call here, so that possible exceptions thrown by it are also caught in the future builder